app/scripts/metamask-controller.js

```diff
@@ -1187,12 +1187,15 @@ export default class MetamaskController extends EventEmitter {
       },
       version,
       // account mgmt
-      getAccounts: async ({ origin }) => {
+      getAccounts: async ({ origin }, suppressUnauthorizedError = true) => {
```

Comment:
Should `suppressUnauthorizedError` be wrapped in an object so that when calling `getAccounts` it's easy to see what the boolean refers to?